python/py-xxhash/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-xxhash
+version             2.0.0
+platforms           darwin
```

Comment:
if no code is compiled, please add `supported_archs noarch`

Reply:
Code is compiled.